lib/ActiveAdmin-Globalize3-inputs.rb

```diff
@@ -1,3 +1,2 @@
 require 'formtastic/globalize_inputs'
-require 'active_admin/globalize_inputs'
```

Comment:
it's not, but if you look at the AA PR linked above they've refactored how the forms are built. This file had previously been included only to wrap around the formtastic file but is no longer necessary.